package.json

```
@@ -152,7 +157,9 @@
         ],
         "displayName": "Get Connections",
         "modelDescription": "Get a list of connections available in the current workspace. ALWAYS call this tool when the user asks about their connections or when connection information is needed - NEVER rely on previous results as connection states change frequently. Any sign-in/sign-out actions are specific to the CCLOUD connection. The LOCAL connection handles Kafka and Schema registry through the local Docker engine API. Any other configurations for Kafka or Schema Registry are done through the DIRECT connections.",
-        "canBeReferencedInPrompt": false,
+        "canBeReferencedInPrompt": true,
+        "toolReferenceName": "getConnections",
+        "userDescription": "Get a list of Kafka connections available in the current workspace.",
```

I'm not sure about the specific Kafka focus here, what do you think about this?:
```suggestion
        "userDescription": "Get a list of Confluent-supported connections available in the current workspace.",
```